lib/src/components/NxTableCell/__tests__/NxTableCell.test.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+import NxTableCell from '../NxTableCell';
+import { faSort, faSortDown, faSortUp } from '@fortawesome/free-solid-svg-icons';
+import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
+
+describe('NxTableCell', function () {
+  it('renders a table with the expected class names', function () {
+    expect(shallow(<NxTableCell className="test"/>)).toMatchSelector('td.nx-cell.test');
```

Comment:
See the way we use `enzymeUtils#getShallowComponent` in most of other other tests to cut down on the repetetiveness of writing these `shallow` invocations

Reply:
done